### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by spencerlyon2**
- Added notes about `Cubic`
- Fixed gridded example so it runs without errors
- Added note about Hessian

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,18 @@ gradient!(g, itp, x, y, ...)
 ```
 where `g` is a pre-allocated vector.
 
+Some interpolation objects support computation of the hessian, which
+can be obtained as
+```jl
+h = hessian(itp, x, y, ...)
+```
+or, if you're evaluating the gradient repeatedly, a somewhat more
```
Comment (tlycken) [inline README.md]: I assume this is meant to read "Hessian" too? :)

**Comment by tlycken** [comment]
Nice, thanks!

**Comment by spencerlyon2** [comment]
Ahh yes good catch! 

Fixed

**Comment by tlycken** [comment]
LGTM - thanks again for taking the time!

**Pull request merged**